AbletonMCP_Remote_Script/handlers/session.py

```
@@ -0,0 +1,1163 @@
+            if val < 0.0 or val > 1.0:
+                raise ValueError("swing_amount must be 0.0-1.0, got {0}".format(val))
+            validated["swing_amount"] = val
+        if clip_trigger_quantization is not None:
+            val = int(clip_trigger_quantization)
+            if val < 0 or val > 13:
+                raise ValueError("clip_trigger_quantization must be 0-13 (Live RecordingQuantization enum), got {0}".format(val))
+            validated["clip_trigger_quantization"] = val
+        if midi_recording_quantization is not None:
+            val = int(midi_recording_quantization)
+            if val < 0 or val > 13:
+                raise ValueError("midi_recording_quantization must be 0-13 (Live RecordingQuantization enum), got {0}".format(val))
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)